packages/js/components/src/product-section-layout/style.scss

```diff
@@ -11,9 +11,9 @@
 			&__body {
 				padding: $gap-large;
 
-				> .components-base-control,
-				> .components-dropdown,
-				> .woocommerce-rich-text-editor {
+				.components-base-control,
```

Comment:
Necessary tweak to allow for the `.woocommerce-product-form__field` container. If it's thought that this could cause issues, we could narrow this selector.